dropwizard-metrics-influxdb/src/main/java/com/izettle/metrics/dw/InfluxDbReporterFactory.java

```diff
@@ -104,6 +104,9 @@
     @NotNull
     private String auth = "";
 
+    @NotNull
+    private TimeUnit timePrecision = TimeUnit.MINUTES;
```

Comment:
@ngong can you change to simply `precision` and add a getter and setter like the other fields? That way it is configurable.